### ISSUE TITLE
difference in the stumpy doc for calculating distance profile

### ISSUE THREAD

**Issue by Ninimama** [TDAmeritrade/stumpy#380]
I think there is a problem in the document. 

In [https://stumpy.readthedocs.io/en/latest/api.html#stumpy.mass](url), there is stumpy.mass(); however, apparently such module does not exist. (I tried it in my work) 

In [https://stumpy.readthedocs.io/en/latest/Tutorial_Pattern_Searching.html?highlight=.core.mass#Summary](url), there is stumpy.core.mass() which is correct.

Is it common to write stumpy.mass()? I checked out the source and .mass() is submodule of .core(). Right? Shouldn't we write it as stumpy.core.mass()?

(Example:
[https://tslearn.readthedocs.io/en/stable/gen_modules/clustering/tslearn.clustering.TimeSeriesKMeans.html](url)
)

**Comment by seanlaw** [comment]
> In https://stumpy.readthedocs.io/en/latest/api.html#stumpy.mass, there is stumpy.mass(); however, apparently such module does not exist. (I tried it in my work)

@Ninimama Great question.

Our documentation reflects the latest **development** version and not the stable release version. In v1.8.0 and before, the `mass` function was not listed in our API. However, due to a recent PR 4ea9fb8368498eba2581aa50a8fc7d782025bb0c (that came after the v1.8.0 release), `stumpy.mass` was added to the public API and it will be available in the upcoming v1.9.0 release. Until then `stumpy.mass` is only available when installed from the latest Github version.

So, if you clone the repo and [install STUMPY from source](https://stumpy.readthedocs.io/en/latest/install.html#from-source), you should be able to:

```
import stumpy
import numpy as np

Q = np.random.rand(10)
T = np.random.rand(100) 

stumpy.mass(Q, T)
```

> In https://stumpy.readthedocs.io/en/latest/Tutorial_Pattern_Searching.html?highlight=.core.mass#Summary, there is stumpy.core.mass() which is correct.

Yes, once v1.9.0 is released then both `stumpy.core.mass` and `stumpy.mass` should BOTH work (stumpy.mass() is just an alias to `stumpy.core.mass()` behind the scenes but `stumpy.core.mass` will be deprecated and we'll need to update the tutorial (I've added a [new issue here](https://github.com/TDAmeritrade/stumpy/issues/381)):

https://github.com/TDAmeritrade/stumpy/blob/f6d0a37d228f5c149cb15ae02559ac005fea6d5b/stumpy/__init__.py#L3

> Is it common to write stumpy.mass()? I checked out the source and .mass() is submodule of .core(). Right? Shouldn't we write it as stumpy.core.mass()?

Hopefully, it is a little bit clearer now. In short, once v1.9.0 comes out, then all of this will be resolved. Sorry for the confusion!

**Comment by Ninimama** [comment]
Got it. Thanks for your thorough response. Appreciate it.
